### ISSUE TITLE
Resolver doesn't resolve custom ObjectType names

### ISSUE THREAD

**Issue by xXKeyleXx** [nestjs/graphql#261]
## I'm submitting a...

<pre><code>
[ ] Regression 
[x] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior
If you rename an ObjectType via the decorator the resolver (`@Resolver`) of nestjs can't resolve the correct schema. The generated gql schema is fine though.

## Expected behavior
The `@Resolver` from `@nestjs/graphql` resolves custom names like the `@Resolver` from `type-graphql`

## Minimal reproduction of the problem with instructions

https://github.com/xXKeyleXx/graphql-minimal-resolver-object-type

### Resolver
```typescript
import { ResolveProperty, Resolver, Root } from '@nestjs/graphql';

@Resolver(of => QuestionSchema)
export class QuestionResolver {
  constructor(
    private questionService: QuestionService,
  ) {
  }

  @ResolveProperty()
  async property(@Root() root: any): Promise<QuestionPropertySchema> {
    ...
  }
}
```
### Typescript Schema
```typescript
import { Field, ID, ObjectType } from 'type-graphql';

@ObjectType('Question')
export class QuestionSchema {
  @Field(type => ID)
  id: string;
}
```

### Generated Schema
```graphql
type Question {
  id: ID!
}
```

## What is the motivation / use case for changing the behavior?
I would like to have different names in the schema and the actual class because I want the classname to correspond to the file(name) structure we chose.

## Environment

<pre><code>+
@nestjs/common: 6.2.3,
@nestjs/core: 6.2.3,
@nestjs/graphql: 6.2.1,
apollo-server-express: 2.5.0
type-graphql: 0.17.4
graphql: 14.3.0 
 
For Tooling issues:
- Node version: v10.15.3
- Platform:  Linux (Docker)
</code></pre>

**Comment by kamilmysliwiec** [comment]
Please, provide a minimal repository which reproduces your issue :)

**Comment by xXKeyleXx** [comment]
https://github.com/xXKeyleXx/graphql-minimal-resolver-object-type

**Comment by gibson** [comment]
If you change import of `Resolver` from `@nest/graphql` to `type-graphql` its normally working.
This is bug of nest wrapper

**Comment by xXKeyleXx** [comment]
I don't remember exactly what it was, but when you switch `Resolver` from `@nest/graphql` to `type-graphql` something else stopped working (I think it was the params of the property resolvers didn't get injected correctly)

**Comment by gibson** [comment]
> I don't remember exactly what it was, but when you switch `Resolver` from `@nest/graphql` to `type-graphql` something else stopped working (I think it was the params of the property resolvers didn't get injected correctly)

Try to set ` allowResolversNotInSchema: true,` in graphql module configuration

**Comment by kamilmysliwiec** [comment]
Using `@Resolver()` directly from `type-graphql` will cause side-effects. This issue is about something else that we have to fix on our side

**Comment by gibson** [comment]
> Using `@Resolver()` directly from `type-graphql` will cause side-effects. This issue is about something else that we have to fix on our side

Im not saying that is resolve issue - this is for vector of searching error

**Comment by gibson** [comment]
But with `allowResolversNotInSchema: true` we have one problem, `@ResolveProperty()` in field resolver context === undefined

**Comment by kamilmysliwiec** [comment]
Fixed in 6.2.4
